File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    
    x_max = 0
    y_max = 0
    
    for i in sizes:
        x, y = sorted(i)
        if x_max < x:
            x_max = x
        if y_max < y:
            y_max = y
            
    answer = x_max * y_max
    
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    sizes.forEach((a) => {
        a.sort((x,y) => x - y)
    });
    
    const maxWidth = Math.max(...sizes.map((a) => a[0]))
    const maxHeight = Math.max(...sizes.map((a) => a[1]))
    
    return maxWidth * maxHeight
}